core-bundle/contao/widgets/ModuleWizard.php

```diff
@@ -81,54 +81,66 @@ public function generate()
 			$modules[$k] = $v;
 		}
 
-		$objRow = $db
-			->prepare("SELECT * FROM " . $this->strTable . " WHERE id=?")
```

Comment:
The changes in this file I would extract into its own PR because this cleanup always makes sense, no matter this PR :)